tests/test_transliterate.py

```diff
@@ -152,3 +152,10 @@ def test_pronunciate(self):
         self.assertIsNotNone(pronunciate("มข.", engine="w2p"))
         self.assertIsNotNone(pronunciate("มช.", engine="w2p"))
         self.assertIsNotNone(pronunciate("jks", engine="w2p"))
+
+    def test_puan(self):
+        self.assertEqual(puan("นาริน"), "นิน-รา")
+        self.assertEqual(puan("นาริน", False), "นินรา")
+        self.assertEqual(puan("สวัสดี"), "สะ-หี-ดวัด")
+        self.assertEqual(puan("สวัสดี", False), "สะหีดวัด")
+        self.assertEqual(puan("สวัสดีครับ"), "สวัสดีครับ")
```

Comment:
It would be good if we have bad input cases (syllable length < 2 or > 3). 
We might even raise an exception for this case.